### PR TITLE
[Testing] Mesh updates: Add setting to limit updates per-frame

### PR DESCRIPTION
**Pull request by paramat**
Not necessarily to be merged, more for testing to see if this helps with FPS jitter.
Try altering the value for meshes processed per frame.
Discussion http://irc.minetest.net/minetest-dev/2017-02-17#i_4814183 onwards.

### CODE REVIEW

**Comment by paramat** [comment]
I still get freezes with this PR, i'm thinking there's another issue and this might not actually help anything, closing for now.

**Comment by lhofhansl** [comment]
I tried that too in my various tests (I had hardcoded a few value back then), didn't do much. :(
I'll try this one.

**Comment by lhofhansl** [comment]
This leads to an annoying delay effect when digging blocks, and - in my case at least - does not help with jitter.

**Comment by lhofhansl** [comment]
I think (at least part of) the stutter stems from work that happens _before_ the update is even queued on the update thread. It's the copying of the blocks data (and it's neighbors, and their neighbors - up to 189 blocks). That happens before the enqueuing, but it's hard getting the synchronization right otherwise.

File: src/defaultsettings.cpp [hunk 1]
```diff
@@ -54,6 +54,7 @@ void set_default_settings(Settings *settings)
 	settings->setDefault("curl_file_download_timeout", "300000");
 	settings->setDefault("curl_verify_cert", "true");
 	settings->setDefault("enable_remote_media_server", "true");
+	settings->setDefault("max_processed_meshes_per_frame", "5");
```
Comment (nerzhul) [inline src/defaultsettings.cpp]: this default limit is ridiculously low :(

**Comment by Fixer-007** [comment]
I would prefer this instead, at least it helps for me and can mitigate worst cases, https://github.com/minetest/minetest/pull/5270

If you are conservative for defaults, sleep can be set to 0 by default, and only some people will increase it to mitigate stutter/jitter.

File: src/defaultsettings.cpp (hunk 1 above)
Comment (paramat) [inline src/defaultsettings.cpp]: Yes it's not a tuned value, it is set to an extreme value to make any improvement more noticeable, to see if this method helps.

**Comment by paramat** [comment]
Those who know the bug better than me find this doesn't help, so closing.

**Pull request closed without merging**